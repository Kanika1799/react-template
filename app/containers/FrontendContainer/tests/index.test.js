/**
 *
 * Tests for FrontendContainer
 *
 *
 */

import React from 'react';
import { renderProvider } from '@utils/testUtils';
import { FrontendContainerTest as FrontendContainer } from '../index';

describe('<FrontendContainer /> container tests', () => {
  let submitSpy;

  beforeEach(() => {
    submitSpy = jest.fn();
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<FrontendContainer dispatchDashboard={submitSpy} />);
    expect(baseElement).toMatchSnapshot();
  });
});
