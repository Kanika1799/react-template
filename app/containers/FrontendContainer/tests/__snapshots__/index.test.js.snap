// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FrontendContainer /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="FrontendContainer__Container-sc-1cnr2ah-0 cVHCgt"
    >
      <div
        class="ant-row"
      >
        <div
          class="ant-col"
        >
          <div
            class="MenubarJs__MenuDiv-sc-9eruv1-0 lautxb"
            data-testid="menubar-js"
          >
            <ul
              class="ant-menu ant-menu-light ant-menu-root ant-menu-vertical"
              role="menu"
              style="background-color: rgb(232, 240, 253); height: 600px;"
            >
              <li
                class="ant-menu-item ant-menu-item-only-child"
                role="menuitem"
                style="height: 70px; margin-left: 0px; margin-top: 20px; padding: 5px;"
              >
                <img
                  src="IMAGE_MOCK"
                />
              </li>
              <li
                class="ant-menu-item ant-menu-item-only-child"
                role="menuitem"
                style="height: 70px; margin-top: 80px; padding: 10px;"
              >
                <img
                  src="IMAGE_MOCK"
                />
              </li>
              <li
                class="ant-menu-item ant-menu-item-only-child"
                role="menuitem"
                style="height: 70px; margin-top: 60px; padding: 10px;"
              >
                <img
                  src="IMAGE_MOCK"
                  width="50"
                />
              </li>
            </ul>
          </div>
        </div>
        <div
          class="ant-col"
        >
          <div
            data-testid="navbar"
          >
            <div
              class="Navbar__HeadingDiv-i5nfb1-0 kGKHKN"
            >
              <div
                class="ant-row"
              >
                <div
                  class="ant-col"
                >
                  <img
                    src="IMAGE_MOCK"
                  />
                </div>
                 
                <div
                  class="ant-col"
                >
                  You have saved  by paying invoices in time. 
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
